### ISSUE TITLE
Better language for the better language page

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#298]
Suggestion from @mitchellevan
Item Number: G153
Part of Item: Description
Comment Type: editorial
Summary of Issue: This page about language needs better language
Comment (Including rationale for any proposed change):
I've quoted some bullet points from the page, and after each quote I've proposed improvements.

(bullet) Use the simplest sentence forms consistent with the purpose of the content. For example, the simplest sentence-form for English consists of Subject-Verb-Object, as in John hit the ball or The Web site conforms to WCAG 2.0.

Improvement: "sentence-form" should be "sentence form", as it is earlier in the same bullet.

Improvement: The phrases "John hit the ball" and "The Web site conforms to WCAG 2.0" should be styled to indicate they are linguistic examples. I'm not sure whether W3C has standard styling for terms and linguistic examples. Wikipedia uses italics.

(bullet) Use sentences that contain no more than two conjunction.

Improvement: The word "conjunction" needs an s on the end. It should be "conjunctions".

(bullet) Remove redundant words, that is, words that do not change the meaning of the sentence.

Improvement: This sentence is ungrammatical, because it's a run-on sentence with a comma splice. Rephrase it, maybe like this: "Remove redundant words. A word is redundant when a sentence has the same meaning with or without the word."

(bullet) Use single nouns or short noun-phrases.

Improvement: Remove the hyphen.

Proposed Change:
See proposed improvements in the Comment field.
